core/src/test/scala/unit/kafka/server/ISRExpirationTest.scala

```diff
@@ -165,6 +162,10 @@ class IsrExpirationTest {
     allReplicas.foreach(r => partition.addReplicaIfNotExists(r))
     // set in sync replicas for this partition to all the assigned replicas
     partition.inSyncReplicas = allReplicas.toSet
+    // make the remote replica read to the end of log
+    for(replica <- partition.assignedReplicas() - leaderReplica)
+      replica.updateLogReadResult(new LogReadResult(FetchDataInfo(new LogOffsetMetadata(10L), MemoryRecords.EMPTY), 10L, 10L, time.milliseconds, -1))
```

Comment:
It's a bit weird to update LogReadResult to a specific offset in a util function. Should the caller do this or at least pass in a required offset?